src/views/v2/latest-past/event/date/featured-icon.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * View: Latest Past View - Single Event Featured Icon
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/v2/latest-past/event/date/featured-icon.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link {INSERT_ARTCILE_LINK_HERE}
+ *
+ * @since 5.1.1
+ *
+ * @var WP_Post $event The event post object with properties added by the `tribe_get_event` function.
+ *
+ * @see tribe_get_event() For the format of the event object.
+ */
+if ( $event->featured ) : ?>
```

Comment:
:no_entry_sign: There must be exactly one blank line after the file comment

<sub>hash: cbde64edd7c0251541072efb9dfd01c1</sub>